cmd/buildah/info.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"runtime"
+	"text/template"
+
+	"github.com/containers/buildah"
+	"github.com/containers/buildah/pkg/parse"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	infoDescription = "Information display here pertain to the host, current storage stats. Useful for the user and when reporting issues"
```

Comment:
Suggest: "The information displayed pertains to the host and current storage statistics which is useful when reporting issues."